core/governance/market.go

```diff
@@ -655,17 +655,18 @@ func validateLPSLAParams(slaParams *types.LiquiditySLAParams) (types.ProposalErr
 	if slaParams == nil {
 		return types.ProposalErrorMissingSLAParams, fmt.Errorf("liquidity provision SLA must be provided")
 	}
-	if slaParams.PriceRange.LessThanOrEqual(num.DecimalZero()) {
+	if slaParams.PriceRange.LessThan(num.DecimalOne()) || slaParams.PriceRange.GreaterThan(num.DecimalFromFloat(100)) {
```

Comment:
This doesn't match the command validation that doesn't allow a price range of 0
```
lppr.IsNegative() || lppr.IsZero() {
    errs.AddForProperty(fmt.Sprintf("%s.price_range", parent), ErrMustBePositive)
}
```

Reply:
because it can't be 0.
``
market.liquidity.priceRange valid values: >0, <=100 
``